drivers/sensor/adxl362/adxl362.c

```diff
@@ -248,16 +306,54 @@ static int axl362_acc_config(struct device *dev, enum sensor_channel chan,
 	return 0;
 }
 
+static int adxl362_attr_set_thresh(struct device *dev, enum sensor_channel chan,
+			    enum sensor_attribute attr,
+			    const struct sensor_value *val)
+{
+	u8_t reg;
+	u16_t threshold = val->val1;
+	size_t ret;
+
+	if (!(SENSOR_CHAN_ACCEL_X ||
+	      SENSOR_CHAN_ACCEL_Y ||
+	      SENSOR_CHAN_ACCEL_Z)) {
```

Comment:
I think you mean to compare `chan` to these values?

Reply:
Oh yes, of course. Thanks!